app/src/test/java/com/github/multimatum_team/multimatum/GroupAdapterTest.kt

```diff
@@ -162,12 +162,14 @@ class GroupAdapterTest {
         @Singleton
         @Provides
         fun provideUserRepository(): UserRepository =
-            MockUserRepository(listOf(
-                UserInfo(id = "Joseph", name = "Joseph"),
-                UserInfo(id = "Louis", name = "Louis"),
-                UserInfo(id = "Florian", name = "Florian"),
-                UserInfo(id = "Léo", name = "Léo"),
-                UserInfo(id = "Val", name = "Val"),
-            ))
+            MockUserRepository(
+                listOf(
+                    UserInfo(id = "Joseph", name = "Joseph"),
+                    UserInfo(id = "Louis", name = "Louis"),
+                    UserInfo(id = "Florian", name = "Florian"),
+                    UserInfo(id = "Léo", name = "Léo"),
+                    UserInfo(id = "Val", name = "Val"),
+                )
```

Comment:
Why am I not part of the list?

Reply:
I did not write this code, it was just formatting, but I copied the same example in my other tests, sorry about that!

Reply:
Nvm, I'll investigate to figured out who's the one that doesn't like me.